File: javascript/7kyu/Return the first M multiples of N.js
// https://www.codewars.com/kata/593c9175933500f33400003e

// Implement a function, multiples(m, n), which returns an array of the first m multiples of the real number n. Assume that m is a positive integer.

function multiples(m, n){
    var arr=[];
    for (var i=1; i<=m; ++i)
      arr.push(n*i);
    return arr;
}

File: java/7kyu/Return the first M multiples of N.java
// https://www.codewars.com/kata/593c9175933500f33400003e

// Implement a function, multiples(m, n), which returns an array of the first m multiples of the real number n. Assume that m is a positive integer.

public class Kata {
  public static int[] multiples(int m, int n) {
    int[] arr = new int[m];
    for(int i = 0; i < arr.length; i++){
      arr[i] = (i + 1) * n;
    }
    return arr;
  }
}